tests/simplex/refinement_03.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2021 by the deal.II authors
+// Copyright (C) 2020 by the deal.II authors
```

Comment:
I get the impression that you're overwriting an existing test here. Please just name your test differently than the one that is pre-existing.